pyensembl/database.py

```diff
@@ -74,7 +74,7 @@ def _all_possible_indices(self, column_names):
         Create list of tuples containing all possible index groups
         we might want to create over tables in this database.
 
-        If a release is missing some column we want to index on,
+        If a genomic database is missing some column we want to index on,
```

Comment:
What do you think of using "genome annotation" to talk about the set of annotation data instead of "genomic database" to avoid confusion with the actual sqlite database we're creating?